pkg/core/system.go

```diff
@@ -236,7 +253,7 @@ func resolveReleaseURLs(filename string) (url.URL, error) {
 	if u.Scheme == "http" || u.Scheme == "https" {
 		return *u, nil
 	}
-	return *u, fmt.Errorf("filename must be file, http or https, got %s", u.Scheme)
+	return *u, fmt.Errorf("filename must be file, http or https, got %s", u.Scheme) // TODO: support file scheme
```

Comment:
if we don't support file, remove it from the error message

Reply:
Done.